Files/Scripts/Options/Ocarina of Time.psm1

```diff
@@ -3516,12 +3540,13 @@ function CreateTabEquipment() {
 
     # HITBOX #
 
```

Comment:
Deku Stick change addition is good, but do not change anything else.

Reply:
Same reasons as before in a previous change within the MM.psm1